api.yaml

```diff
@@ -273,6 +352,142 @@ endpoints:
               statusCode: "400"
           type: http
           uri: "https://${stageVariables.enterprise_host}/enterprise/api/v1/catalogs/"
+    newEnterpriseCatalogs:
```

Comment:
P.S. This is a generated file, so if I change api-compact.yml, this will update through the yamllint.com generator.